snuba/state/__init__.py

```diff
@@ -235,7 +236,7 @@ def record_query(query_metadata: Mapping[str, Any]) -> None:
 
         kfk.poll(0)  # trigger queued delivery callbacks
         kfk.produce(
-            settings.QUERIES_TOPIC,
+            Topic.QUERYLOG.value,
```

Comment:
Can't we override the name of this one ?

Reply:
Fixed to pick up the right name